server/src/test/java/org/elasticsearch/cluster/ClusterHealthIT.java

```diff
@@ -250,15 +259,35 @@ public void clusterStateProcessed(String source, ClusterState oldState, ClusterS
     }
 
     @Test
+    @UseRandomizedSchema(random = false)
     public void testHealthOnMasterFailover() throws Exception {
         final String node = internalCluster().startDataOnlyNode();
+        boolean withIndex = true; //randomBoolean();
```

Comment:
currently always failing on true, cluster is always YELLOW in the last assertion